site/sfguides/src/vhol_iceberg_snowflake_open_catalog/vhol_iceberg_snowflake_open_catalog.md

```diff
@@ -0,0 +1,575 @@
+    Leave the rest of the configuration default
+    Select Create bucket.
+- Once it’s created successfully, on the next screen (S3 buckets list), click the name of the S3 bucket you created
+- On the right, click “Create folder”
+- On the next screen:
+    For “Folder name”, enter “snowflake-managed-iceberg”
+    Leave the rest of the configurations as default
+    Click “Create folder”
+- Click the checkbox next to the folder name you just created. Then click “Copy S3 URI”. Paste this value in the HOL spreadsheet.
+
+<!-- ------------------------ -->
+
+## Create an IAM policy that grants access to your S3 location
+
+- In the AWS Management Console, search or browse to IAM
+- In the navigation pane on the left under “Access management”, select “Account settings”
+- Under Security Token Service (STS), in the Endpoints list, find the line for **US West-2**. If the STS status is inactive, set the toggle to Active.
+
+![sts endpoints page](assets/img03.PNG)
+
+- In the navigation pane on the left under “Access management”, select “Policies”
+- Select “Create Policy” in the top right
+- On the next screen:
+    For Policy editor, select JSON.
+    Replace the pre-populated JSON contents of the Policy Editor with the following JSON. Note don’t click save after pasting yet.
+
+
+```json
+{
+      "Version": "2012-10-17",
+      "Statement": [
+            {
+               "Effect": "Allow",
+               "Action": [
+                  "s3:PutObject",
+                  "s3:GetObject",
+                  "s3:GetObjectVersion",
+                  "s3:DeleteObject",
+                  "s3:DeleteObjectVersion"
+               ],
+               "Resource": "arn:aws:s3:::<my_bucket>/*"
+            },
+            {
+               "Effect": "Allow",
+               "Action": [
+                  "s3:ListBucket",
+                  "s3:GetBucketLocation"
+               ],
+               "Resource": "arn:aws:s3:::<my_bucket>",
+               "Condition": {
+                  "StringLike": {
+                        "s3:prefix": [
+                           "*"
+                        ]
+                  }
+               }
+            }
+      ]
+   }
+```
+
+    Before clicking save, replace “<my_bucket>” with the name of the bucket you specified in step 2.e.i (i.e., deconnect-hol-demo-<name>) in both places. Note that the value you replace with this should just be the name of the bucket (“deconnect-hol-demo-<name>”) and not include the protocol (i.e., “s3://”)
+    In the bottom right, click “Next”
+    For “Policy name”, enter “deconnect-hol-demo-policy”
+    Scroll down, and in the bottom right click “Create policy”
+
+<!-- ------------------------ -->
+
+## Create an IAM role to grant privileges on your S3 bucket
```

Comment:
@vinodhini-sd Shorten and title case the step label so the left-nav looks cleaner